de.bforartists.Bforartists.yaml

```diff
@@ -0,0 +1,176 @@
+      - --enable-libvpx
+      - --enable-libx264
+      - --enable-zlib
+      - --disable-libxcb
+      - --disable-lzma
+      - --disable-programs
+      - --disable-network
+      - --disable-protocols
+      - --enable-protocol=file
+      - --disable-devices
+      - --enable-muxer=avi
+      - --enable-muxer=h264
+      - --enable-muxer=mov
+      - --enable-muxer=mp4
+      - --enable-muxer=ogg
+      - --enable-muxer=webm
+      - --enable-demuxer=avi
+      - --enable-demuxer=h264
+      - --enable-demuxer=mov
+      - --enable-demuxer=mp3
+      - --enable-demuxer=ogg
+      - --enable-demuxer=wav
+      - --enable-parser=h264
+      - --enable-parser=vorbis
+      - --enable-encoder=aac
+      - --enable-encoder=libtheora
+      - --enable-encoder=libvorbis
+      - --enable-encoder=libvpx_vp8
+      - --enable-encoder=libvpx_vp9
+      - --enable-encoder=libx264
+      - --enable-encoder=mpeg4
+      - --enable-decoder=aac
+      - --enable-decoder=h264
+      - --enable-decoder=libvorbis
+      - --enable-decoder=libvpx_vp8
+      - --enable-decoder=libvpx_vp9
+      - --enable-decoder=mp3
+      - --enable-decoder=mpeg4
+      - --enable-decoder=pcm_s16le
+      - --enable-decoder=theora
+    sources:
+      - type: archive
+        url: https://www.ffmpeg.org/releases/ffmpeg-6.0.tar.xz
+        sha256: 57be87c22d9b49c112b6d24bc67d42508660e6b718b3db89c44e47e289137082
+    cleanup:
+      - /lib/codecs/include
+      - /lib/codecs/lib/pkgconfig
+      - /lib/codecs/share
+  
+  # Bforartists itself
+  - name: bforartists
+    buildsystem: simple
+
+    # All commands executed that ultimately
+    # install Bforartists to the Flatpak 
+    # runtime. This includes desktop files/icons,
+    # as well as the metadata file used for Flathub
+    build-commands:
+      - mv Bforartists*Linux* bforartists
+      - install -Dm755 bforartists.sh /app/bin/bforartists
+      - mv bforartists /app/bforartists
+      - install -Dm644 /app/bforartists/bforartists.desktop /app/share/applications/$FLATPAK_ID.desktop
+      - desktop-file-edit --set-icon=${FLATPAK_ID} /app/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm644 /app/bforartists/bforartists.svg /app/share/icons/hicolor/scalable/apps/$FLATPAK_ID.svg
+      - install -Dm644 /app/bforartists/bforartists-symbolic.svg /app/share/icons/hicolor/symbolic/apps/$FLATPAK_ID-symbolic.svg
+      - cd /app/bforartists/*.*/datafiles && mv locale /app/share/locale && ln -sf /app/share/locale locale
+      - install -Dm644 $FLATPAK_ID.metainfo.xml /app/share/metainfo/$FLATPAK_ID.metainfo.xml
+
+    # Bforartists tar archive, retrived from
+    # GitHub releases and verified with a sha256 
+    # checksum. For every update, the URL and checksum 
+    # need to be updated. For the checksum, it is as simple
+    # as downloading the tar archive and running
+    # sha256sum <bforartists tar archive>
+    #
+    # In addition, the version for x-checker-data needs be updated
```

Comment:
The point of x-checker-data is to save you the trouble and have something else detect the new version.

Reply:
So something like this?
```yaml
x-checker-data:
  type: anitya
  project-id: 359756
  stable-only: true
  versions:
    '>=': '3.5.1'
  url-template: https://github.com/Bforartists/Bforartists/releases/download/v$version/Bforartists-$version0-$version1-$version2-Linux.tar.xz
```